stats.go

```diff
@@ -366,15 +366,19 @@ func (s *statStore) NewCounter(name string) Counter {
 		return c
 	}
 
-	c = &counter{}
```

Comment:
This changes the behaviour of New though. It will not create a new Counter everytime now. Is that intended?

Reply:
Never mind, I see it is.